frontend/src/features/admin-form/responses/AdminSubmissionsService.ts

```diff
@@ -24,3 +33,22 @@ export const countFormSubmissions = async ({
   }
   return ApiService.get(queryUrl).then(({ data }) => data)
 }
+
+/**
+ * Get of submissions metadata for a given form
+ * @param formId fomID to retrieve metadata on
+ * @returns The metadata of the form
+ */
+export const getFormSubmissionsMetadata = async (
+  formId: string,
+): Promise<StorageModeSubmissionMetadataList> => {
+  const queryUrl = `${ADMIN_FORM_ENDPOINT}/${formId}/submissions/metadata?page=1` // To be generalized in later PRs
+  return ApiService.get(queryUrl).then(({ data }) => data)
+}
+
+/**
+ * Terminates all workers
+ * @param workers an array of workers
+ */
+export const killWorkers = (workers: DecryptionWorker[]): void =>
```

Comment:
should this function be in the hook itself since that's the only location its used?